asterisk.conf.diff

```diff
@@ -0,0 +1,32 @@
+diff --git a/asterisk-13/asterisk.conf b/asterisk-13/asterisk.conf
```

Comment:
We might be better off with `make samples`. I was avoiding that, b/c I'm a "start Asterisk from nothing" sort of guy, but the sample configs probably make more sense for general use.